spikeinterface/sortingcomponents/peak_selection.py

```diff
@@ -51,15 +52,27 @@ def select_peaks(peaks, method='uniform', seed=None, **method_kwargs):
                     Total number of peaks to select
                 * peaks_locations: array
                     The locations of all the peaks, computed via localize_peaks
-    
+
```

Comment:
@pauladkisson can you add `selected_indices` (only returned when `return_indices` is `True`)?

Reply:
Done!